build/common/installer/scripts/tomlparser-prom-customconfig.rb

```diff
@@ -149,6 +151,10 @@ def populateSettingValuesFromConfigMap(parsedConfig)
           urls = parsedConfig[:prometheus_data_collection_settings][:cluster][:urls]
           kubernetesServices = parsedConfig[:prometheus_data_collection_settings][:cluster][:kubernetes_services]
 
+          if fieldPass.nil? && fieldDrop.nil? && urls.nil? && kubernetesServices.nil?
```

Comment:
We should also need to turn off telegraf in RS when there is no configmap  applied or configmap with no cluster config settings, right?